apps/admin-server/src/pages/index.tsx

```diff
@@ -1,11 +1,23 @@
 import { Inter } from 'next/font/google';
 import { signIn, signOut, useSession } from 'next-auth/react';
 import { useRouter } from 'next/navigation';
+import { useEffect } from 'react';
 
 const inter = Inter({ subsets: ['latin'] });
 
 export default function Home() {
   const { data } = useSession();
+
+  // dit moet in elke pagina, niet alleen de home
```

Comment:
Bart heeft dit als het goed is verplaatst naar de sidebar zodat de useEffect daar altijd aangeroepen wordt. Deze useEffect kan er dus uit worden gehaald. Bart heeft de changes in deze PR overgenomen in zijn pr 116 dus deze zou eventueel verwijderd kunnen worden